### ISSUE TITLE
Digest and Signature in message

### ISSUE THREAD

**Issue by SunFulong** [wamp-proto/wamp-proto#447]
Will we plan to add field to place Message Digest and Signature in some message like these:

https://www.rfc-editor.org/rfc/rfc3230
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-10
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-16

**Comment by oberstet** [comment]
yes, individual authenticated MACs of individual messages are part of https://github.com/wamp-proto/wamp-proto/pull/419
